src/queries/sc/domain-hints.sql

```diff
@@ -3,7 +3,7 @@
 --- Cache-Control: max-age=3600
 --- timezone: UTC
 --- device: all
---- domainkey: secret
+--- domain key: secret
```

Comment:
There's no space between `domain` and `key`

Reply:
Correct @dbrrt, this was only a test